File: DarkOrbit 7.5.3/Net/netty/commands/PetGearRemoveCommand.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Ow.Utils;

namespace Ow.Net.netty.commands
{
    class PetGearRemoveCommand
    {
        public const short ID = 8050;

        public static byte[] write(PetGearTypeModule gearType, int level, int amount)
        {
            var param1 = new ByteArray(ID);
            param1.write(gearType.write());
            param1.writeInt(level);
            param1.writeInt(amount);
            return param1.ToByteArray();
        }
    }
}

File: DarkOrbit 9.0/DarkOrbit 9.0 Emulator/src/simulator/netty/serverCommands/PetGearRemoveCommand.java
package simulator.netty.serverCommands;

import java.io.DataOutputStream;
import java.io.IOException;

import simulator.netty.ServerCommand;

/**
Created by LEJYONER on 21/09/2017.
*/

public class PetGearRemoveCommand
        implements ServerCommand {

    public static int ID       = 30680;
    public        PetGearTypeModule gearType;
    public        int level  = 0;
    public 		  int amount = 0;

    public PetGearRemoveCommand(PetGearTypeModule param1, int param2, int param3) {
        this.gearType = param1;
        this.level = param2;
        this.amount = param3;
    }

    public void write(DataOutputStream param1) {
        try {
            param1.writeShort(ID);
            this.writeInternal(param1);
        } catch (IOException e) {
        }
    }

    protected void writeInternal(DataOutputStream param1) {
        try {
            this.gearType.write(param1);
            param1.writeInt(this.level << 3 | this.level >>> 29);
            param1.writeShort(9304);
            param1.writeInt(this.amount >>> 15 | this.amount << 17);
            } catch (IOException e) {
        }
    }
}